scripts/testEnv.sh

```diff
@@ -54,6 +54,9 @@ export MAXIMUM_WAIT=${MAXIMUM_WAIT:-30}
 # o LISTENER2_PORT - port where a second listening (test) application listens for connections
 #
 export CB_PORT=${CB_PORT:-9999}
+export CB_A_PORT=${CB_A_PORT:-1030}
```

Comment:
Same comments for databases, logfiles and pidfiles. That way, you can save to change the testEnv.sh file for this PR.